test/fixtures/simple-esm/helper/read.js

```diff
@@ -14,7 +14,7 @@ import path from 'path';
 
 export default () => {
   try {
-    const hello = path.resolve(__rootdir, 'files', 'hello.txt');
+    const hello = path.resolve(process.cwd(), 'files', 'hello.txt');
```

Comment:
files must be read from `process.cwd()` - to either match the temporary directory locally or the extraction directory remotely